etc/conf/base.py

```diff
@@ -23,11 +23,11 @@ def unsupported(platform):
     arch = '32'
 
 sys_platform = str(sys.platform).lower()
-if 'linux' in sys_platform:
+if sys_platform.startswith('linux'):
     os = 'linux'
-elif 'win32' in sys_platform:
+elif sys_platform == 'win32':
```

Comment:
There may be some risk attached to changing the tests here... Do you mind leaving these as is?